software/webapp/src/main/java/brooklyn/entity/webapp/nodejs/NodeJsWebAppSshDriver.java

```diff
@@ -152,6 +152,8 @@ public void launch() {
             appCommandLine = appCommand + " " + appFile;
         }
 
+        // Ensure global NPM modules are on Node's path.
+        commands.add("export NODE_PATH=\"$NODE_PATH:$(npm root -g)\"");
```

Comment:
Looks like it's bad practice to rely on global modules, but not much to do if it's needed by real-world apps.
https://docs.npmjs.com/misc/faq#i-installed-something-globally-but-i-can-t-require-it